rendezvous.php

```diff
@@ -206,6 +208,7 @@ function book_form2($ren_ses_id)
 
   if($_SERVER['REQUEST_METHOD'] == 'POST')
   {
+    validateToken();
```

Comment:
Similarly with the other comment, if you check all `POST` requests, it makes sense to include this check once, in the beginning of the file, and **before ALL** calls to the function that generates new tokens.

Reply:
I'll make sure to fix it.